x-pack/plugins/security_solution/public/common/components/risk_score/risk_score_onboarding/utils.test.ts

```diff
@@ -0,0 +1,311 @@
+  restartRiskScoreTransforms,
+  uninstallLegacyRiskScoreModule,
+} from './utils';
+
+jest.mock('../../../../risk_score/containers/onboarding/api');
+
+const mockHttp = {
+  post: jest.fn(),
+} as unknown as HttpSetup;
+const mockSpaceId = 'customSpace';
+const mockTimerange = {
+  from: 'startDate',
+  to: 'endDate',
+};
+const mockRefetch = jest.fn();
+describe('installHostRiskScoreModule', () => {
+  beforeAll(async () => {
+    await installHostRiskScoreModule({
+      http: mockHttp,
+      refetch: mockRefetch,
+      spaceId: mockSpaceId,
+      timerange: mockTimerange,
+    });
+  });
+
+  afterAll(() => {
+    jest.clearAllMocks();
+  });
+
+  it(`Create script: ml_${RiskScoreEntity.host}riskscore_levels_script_${mockSpaceId}`, async () => {
+    expect((api.createStoredScript as jest.Mock).mock.calls[0][0].options).toMatchSnapshot();
+  });
+
+  it(`Create script: ml_${RiskScoreEntity.host}riskscore_init_script_${mockSpaceId}`, async () => {
+    expect((api.createStoredScript as jest.Mock).mock.calls[1][0].options).toMatchSnapshot();
+  });
+
+  it(`Create script: ml_${RiskScoreEntity.host}riskscore_map_script_${mockSpaceId}`, async () => {
+    expect((api.createStoredScript as jest.Mock).mock.calls[2][0].options).toMatchSnapshot();
+  });
+
+  it(`Create script: ml_${RiskScoreEntity.host}riskscore_reduce_script_${mockSpaceId}`, async () => {
+    expect((api.createStoredScript as jest.Mock).mock.calls[3][0].options).toMatchSnapshot();
+  });
+
+  it(`Create IngestPipeline: ml_${RiskScoreEntity.host}riskscore_ingest_pipeline_${mockSpaceId}`, async () => {
+    expect((api.createIngestPipeline as jest.Mock).mock.calls[0][0].options).toMatchSnapshot();
+  });
+
+  it(`Create Index: ml_${RiskScoreEntity.host}_risk_score_${mockSpaceId}`, async () => {
+    expect((api.createIndices as jest.Mock).mock.calls[0][0].options).toMatchSnapshot();
+  });
+
+  it(`Create Index: ml_${RiskScoreEntity.host}_risk_score_latest_${mockSpaceId}`, async () => {
+    expect((api.createIndices as jest.Mock).mock.calls[1][0].options).toMatchSnapshot();
+  });
+
+  it(`Create Transform: ml_${RiskScoreEntity.host}riskscore_pivot_transform_${mockSpaceId}`, async () => {
+    expect((api.createTransform as jest.Mock).mock.calls[0][0].options).toMatchSnapshot();
+  });
+
+  it(`Create Transform: ml_${RiskScoreEntity.host}riskscore_latest_transform_${mockSpaceId}`, async () => {
+    expect((api.createTransform as jest.Mock).mock.calls[1][0].options).toMatchSnapshot();
+  });
+
+  it(`Start Transforms`, () => {
+    expect((api.startTransforms as jest.Mock).mock.calls[0][0].transformIds).toMatchSnapshot();
+  });
+
+  it(`Create Hosts dashboards`, () => {
+    expect(
+      (bulkCreatePrebuiltSavedObjects as jest.Mock).mock.calls[0][0].options.templateName
+    ).toEqual(`${RiskScoreEntity.host}RiskScoreDashboards`);
+  });
+
+  it('Refresh module', () => {
+    expect(mockRefetch).toBeCalled();
+  });
+});
+
+describe(`installUserRiskScoreModule`, () => {
```

Comment:
another place we could use `describe.each`

Reply:
I made `installUserRiskScoreModule ` and `installHostRiskScoreModule ` private functions, consumed by `installRiskScoreModule`, we pass riskScoreEntity to decide which module to install.
Users rick score has one step less than hosts risk score, so the calls' count does not the same between the two tests, so I left this one separate and combined rest of the tests.